calico_node/filesystem/etc/calico/confd/config/custom_filters.cfg

```diff
@@ -0,0 +1,5 @@
+# Generated by confd
```

Comment:
@neiljerram it's possible to use `exists` in templates. But this errors comes from `.toml` config file, not from template. AFAIK, it's not possible to bypass this in toml.